srcpkgs/visual-studio-code-bin/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'visual-studio-code-bin'
+pkgname=visual-studio-code-bin
+version=1.24.1
+revision=1
+only_for_archs="x86_64"
+wrksrc="${pkgname}-${version}"
```

Comment:
This is the default, so you can leave it out

Reply:
Which parameter is default?